tools/functional-tester/etcd-runner/main.go

```diff
@@ -310,10 +320,24 @@ func runWatcher(eps []string) {
 						return
 					}
 				}
-			}(prefixes[i])
+			}(roundPrefix + "-" + prefixes[i])
 		}
 	}
 	wg.Wait()
+
+	// cancel all watch channels
+	cancelc()
+
```

Comment:
@heyitsanthony yes may be you are write since i wrote a simple example and it was working when i was not closing on the client. So what do you suggest, in this code shall we move closing of the client in the end?